pkgs/top-level/all-packages.nix

```diff
@@ -2464,7 +2464,8 @@ in
   ddrutility = callPackage ../tools/system/ddrutility { };
 
   deluge = callPackage ../applications/networking/p2p/deluge {
-    pythonPackages = python2Packages;
+    pythonPackages = python3Packages;
+    libtorrentRasterbar = libtorrentRasterbar.override { python = python3; };
```

Comment:
probably better to perform this override inside the deluge expression to ensure the same python is always used.

Reply:
Actually, this package is already overridden in `python-packages.nix`!